app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/individual/HiddenHealthRecordAdapter.kt

```diff
@@ -0,0 +1,36 @@
+package ca.bc.gov.bchealth.ui.healthrecord.individual
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import ca.bc.gov.bchealth.databinding.ItemLoginForHiddenRecordsBinding
+
+class HiddenHealthRecordAdapter(private val loginClickListener: ItemClickListener) :
+    RecyclerView.Adapter<HiddenHealthRecordAdapter.ViewHolder>() {
```

Comment:
Please use ListAdapter instead of RecyclerView.Adapter

Reply:
Done